client/state/data-layer/wpcom/sites/media/test/index.js

```diff
@@ -52,11 +52,9 @@ describe( 'media request', () => {
 	} );
 
 	test( 'should dispatch FAILURE action when request fails', () => {
```

Comment:
I realize those tests were added before and not as part of our work here, but I wanted to point out that they seem to be pretty redundant. Maybe we can clean them up, what do you think?